varinfo/cmr_search.py

```diff
@@ -62,13 +63,41 @@ def get_granules(concept_id: str = None,
     return granule_response
 
 
-def get_granule_link(query_response: list) -> str:
+def get_granule_link(granule_response: list) -> str:
     ''' Get the granule download link from CMR
     '''
     granule_link = next((link['href']
-                         for link in query_response[0].get('links', [])
+                         for link in granule_response[0].get('links', [])
                          if link['rel'].endswith('/data#') and 'inherited' not in link), None)
 
     if granule_link is None:
-        raise MissingGranuleDownloadLinks(query_response)
+        raise MissingGranuleDownloadLinks(granule_response)
     return granule_link
+
+
+def get_granule_name(granule_response: list) -> str:
```

Comment:
This really just a helper function for getting the granule filename, so we don't have to go through the `granule_response`